index.less

```diff
@@ -1,6 +1,8 @@
 // Follow the order in the readme 
-@import "reset"; 
-@import "global"; 
-@import "navigation"; 
-@import "footer"; 
-@import "home-page"; 
\ No newline at end of file
+@import "variables.less";
```

Comment:
this is breaking, because variables.less does not exist. Also mixin.less

Reply:
Everything was working fine for me yesterday. I had the watch compiler working on everything I did.

Reply:
Also I had live server working good too and I had no problems.

Reply:
Oh, I see you copied the less files to less folder.  How about we delete old less files. Then everything shall be good.